content/release-10-14-0/devices-10-14-0-bundle/10-14-0-268.md

```diff
@@ -0,0 +1,70 @@
+---
+weight: 26
+title: Release 10.14.0.268
+layout: bundle
+---
+
+<!--10.14.0.257 - 10.14.0.268-->
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device Management</td>
+<td>The performance of the advanced-software-mgmt microservice has been improved.</td>
+<td>DM-1251</td>
+</tr>
+
+</tbody></table></div>
+
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device Management</td>
+<td>The groups list on the <b>Groups</b> page now offers the option to filter by the grid's columns.</td>
+<td>DM-1836</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>By default, the LWM2M object-resource mappings of the Management tenant that are configured using the LWM2M device protocols were not inherited by the subtenants. With this change, the LWM2M object-resource mappings defined in the Management tenant are applied to all its subtenants by default. The default mappings from the Management tenant can also be overwritten by uploading the LWM2M device protocol for the LWM2M object on the subtenant level as described in <a href="https://cumulocity.com/guides/protocol-integration/lwm2m/#device-protocols" class="no-ajaxy">LWM2M > LWM2M device protocols</a> in the <i>Protocol integration guide</i>. </td>
+<td>DM-1778</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>While updating the security info for PSK devices in the database, the LWM2M agent was internally deleting the old security managed object and creating a new one. Due to the asynchronous deletion of managed objects in the Cumulocity IoT platform, the agent intermittently failed to update the security info. With the current changes, the LWM2M agent updates the required security details in the same managed object instead of removing it first.</td>
```

Comment:
```suggestion
<td>While updating the security info for PSK devices in the database, the LWM2M agent internally deleted the old security managed object and created a new one. Due to the asynchronous deletion of managed objects in the Cumulocity IoT platform, the agent intermittently failed to update the security info. With the current changes, the LWM2M agent updates the required security details in the same managed object instead of removing it first.</td>
```